.vscode/settings.json

```diff
@@ -1,5 +1,7 @@
 {
     "githubPullRequests.ignoredPullRequestBranches": [
         "master"
-    ]
+    ],
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true

```

Comment:
This repo not use pylint.  Instead, we use ruff that includes the pylint rules.
https://github.com/TheAlgorithms/Python/blob/d31750adece86ebf39a09dd3adb2039098f58586/pyproject.toml#L111